conf.py

```diff
@@ -16,7 +16,7 @@
 # BASE_URL = "https://getnikola.com/"
 BLOG_EMAIL = "no-reply@legacysurvey.org"
 BLOG_DESCRIPTION = "Legacy Survey"
-LOGO_URL = "/files/logo_tiny.jpg"
+LOGO_URL = "/files/logo_very_tiny.jpg"
```

Comment:
This one I'm concerned about, I had to tweak the CSS corresponding to this file after updating the jQuery + Bootstrap framework.  We might need to just test this on several browsers.

Reply:
PS, the custom.css file assumes a specific `height` for the image in the navbar, but `logo_tiny` and `logo_very_tiny` do not have the same height.

Reply:
I reverted back to `logo_tiny.jpg` and cleaned up the toolbar accordingly. My only reason for using `logo_very_tiny.jpg` was to put an extra column in the toolbar, but that was a stylistic choice and hardly critical.